DataScience/Experimentation.py

```diff
@@ -200,14 +200,19 @@ def parse_cb_types(val):
         raise argparse.ArgumentTypeError('Input "{}" is an invalid cb_types input string - Valid cb_types are mtr, dr, ips'.format(val))
     return cb_types
 
-def generate_predictions_files(log_fp, policies):
+def create_prediction_command(policy_command, log_type):
+    if log_type == 'ccb':
+        return policy_command.replace('--ccb_explore_adf --epsilon 0', '--ccb_explore_adf --epsilon 0.2')
```

Comment:
just curious: why one has `--epsilon 0` but the other does not have it.

Reply:
cb has `cb_adf` and `cb_explore_adf`, when sweeping the commands to find the best policies, we used `cb_adf` and for prediction we used `cb_explore_adf --epsilon 0.2`

ccb only has `ccb_explore_adf`, when sweeping we used `ccb_explore_adf --epsilon 0` to do exploitation and for prediction we used `ccb_explore_adf --epsilon 0.2`